client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx

```diff
@@ -1,19 +1,21 @@
 import { ProgressBar } from '@automattic/components';
 import { useTranslate } from 'i18n-calypso';
 import { StepNavigationLink } from 'calypso/../packages/onboarding/src';
+import Badge from 'calypso/components/badge';
 import WordPressLogo from 'calypso/components/wordpress-logo';
 import { NavigationControls } from 'calypso/landing/stepper/declarative-flow/internals/types';
 import { useFlowParam } from 'calypso/landing/stepper/hooks/use-flow-param';
 import { useSite } from 'calypso/landing/stepper/hooks/use-site';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { ResponseDomain } from 'calypso/lib/domains/types';
 import Checklist from './checklist';
 import { getArrayOfFilteredTasks, getEnhancedTasks, isTaskDisabled } from './task-helper';
 import { tasks } from './tasks';
 import { getLaunchpadTranslations } from './translations';
 import { Task } from './types';
 
 type SidebarProps = {
-	sidebarURL: string | null;
+	sidebarDomainObject: ResponseDomain;
```

Comment:
`NB`: I'm unsure about including types like `object`, `array`, or `string` in variable names, especially since we have typescript to give us information about the prop

Does `sidebarDomain` work on its own? If not, could we think of better alternatives?

Reply:
I updated `sidebarDomainObject` to `sidebarDomain`.